ui/main/src/app/modules/cards/components/detail/detail.component.ts

```diff
@@ -168,6 +181,16 @@ export class DetailComponent implements OnChanges, OnInit, OnDestroy, AfterViewC
         this.markAsRead();
     }
 
+    get lttdExpired(): boolean {
+        this.store.select(fetchLightCard(this.card.id)).pipe(take(1))
+        .subscribe((lightCard: LightCard) => {
+            this.lttd = lightCard.lttd;
+        });
+
+        return this.lttdExpiredIsTrue || Math.floor((this.lttd - new Date().getTime()) / 1000) <= 0;
```

Comment:
To get the lttd, you do not need to access light card , you can find the value in the current card : 
`card.lttd`
 so the code for this method will be just : 
`return this.lttdExpiredIsTrue || Math.floor((card.lttd - new Date().getTime()) / 1000) <= 0`

Reply:
ok

Reply:
I agree to recover lttd from the card but must add the nullity test on lttd